src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt

```diff
@@ -90,10 +104,14 @@ fun main(args: Array<String>) = mainBody {
 class CosmasServerArgs(parser: ArgParser) {
     val port: Int by parser.storing("--port",
             help = "choose port that server will listen to, 50051 by default") { toInt() }.default { 50051 }
-    val bucket: String by parser.storing("--bucket",
-            help = "choose Google Cloud bucket for files storing, \"papeeria-interns-cosmas\" by default").default { "papeeria-interns-cosmas" }
+    val bucket: String? by parser.storing("--bucket",
+            help = "choose Google Cloud bucket for files storing, \"papeeria-interns-cosmas\" by default").default { null }

```

Comment:
So it is not `papeeria-interns-cosmas` by default.

Reply:
done